tests/secrets/mkdir.test.ts

```diff
@@ -56,6 +56,20 @@ describe('commandMkdir', () => {
       expect(stat.isDirectory()).toBeTruthy();
     });
   });
+  test('should fail when provided only the vault name', async () => {
+    const vaultName = 'vault' as VaultName;
+    const vaultId = await polykeyAgent.vaultManager.createVault(vaultName);
+    command = ['secrets', 'mkdir', '-np', dataDir, vaultName];
+    const result = await testUtils.pkStdio([...command], {
```

Comment:
This needs to change to `command` instead of `[...command]`. This actually needs to be consistent throughout the secrets tests.

Reply:
Secrets env and secrets mkdir had this issue. That has now been resolved. This has not been addressed for the vault tests, as they are out of scope for this.